Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM datasetteproject/datasette:0.54
+FROM datasetteproject/datasette:latest
 
-RUN apt-get update && apt-get -y upgrade && \
-  apt-get install --no-install-recommends -y \
-  build-essential \
-  unzip \
-  && \
-  apt-get clean
+RUN apt-get update 
+# RUN apt-get -y upgrade
```

Comment:
Some thoughts on why this might be failing:

- `apt-get update` should _"install the newest versions of all packages currently installed on the system"_ [(ref)](https://linux.die.net/man/8/apt-get)
- The `-y` flag should provide _"Automatic yes to prompts"_
  - However, there's a caveat: _"If an undesirable situation, such as changing a held package or removing an essential package, occurs then apt-get will abort."_
  - (Held package = [package that's locked to a specific version](https://www.cyberciti.biz/faq/apt-get-hold-back-packages-command/), I think)

So perhaps the latest datasette image has some packages on lock, so the "yes, please upgrade those packages" command causes an error. 🤷 

Anyway, I'm not sure if removing `apt-get upgrade` is a good idea in the long run, but this seems to work, at least locally. I'm curious to see how this builds on GitHub Actions.